packages/bundler-plugin-core/package.json

```diff
@@ -42,7 +42,8 @@
     "find-up": "5.0.0",
     "glob": "9.3.2",
     "magic-string": "0.27.0",
-    "unplugin": "1.0.1"
+    "unplugin": "1.0.1",
+    "webpack-sources": "^3.2.3"
```

Comment:
m: should we hard pin? Also this doesn't change `yarn.lock`?

Reply:
good point. Added. The yarn lock didn't change because unplugin itself depended on the exact same version.